### PR TITLE
blobfuse: init at 1.0.1-RC-Preview

### PR DESCRIPTION
**Pull request by jbgi**

### CODE REVIEW

**Comment by jbgi** [comment]
@Mic92: updated to latest tag, removing all patches.

File: pkgs/tools/filesystems/blobfuse/default.nix [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  version = "1.0.1-RC-Preview";
+
+  src = fetchFromGitHub {
+    owner  = "Azure";
+    repo   = "azure-storage-fuse";
+    rev    = "v${version}";
+    sha256 = "143rxgfmprir4a7frrv8llkv61jxzq50w2v8wn32vx6gl6vci1zs";
+  };
+
+  buildInputs = [ cmake curl gnutls libgcrypt libuuid fuse ];
```
Comment (adisbladis) [inline pkgs/tools/filesystems/blobfuse/default.nix]: `cmake` should be in `nativeBuildInputs` as it needs to be executed on the build platform.

**Review by adisbladis — CHANGES_REQUESTED** [review]

File: pkgs/tools/filesystems/blobfuse/default.nix [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  nativeBuildInputs = [ pkgconfig ];
+
+  meta = with stdenv.lib; {
+    description = "Mount an Azure Blob storage as filesystem through FUSE";
+    license = licenses.mit;
+    platforms = platforms.linux;
```
Comment (Mic92) [inline pkgs/tools/filesystems/blobfuse/default.nix]: Do you want to maintain this package?  https://nixos.wiki/wiki/Nixpkgs#Becoming_a_Nixpkgs_maintainer

**Comment by jbgi** [comment]
@adisbladis: fixed.

**Comment by Mic92** [comment]
@GrahamcOfBorg build blobfuse

**Pull request merged**